tests/ut/backends/test_redis.py

```diff
@@ -6,7 +6,7 @@
 from aiocache import RedisCache
 from aiocache.base import BaseCache
 from aiocache.serializers import JsonSerializer
-from aiocache.backends.redis import RedisBackend, conn
+from aiocache.backends.redis import RedisBackend, conn, AIOREDIS_BEFORE_ONE
 
 
 class FakeRedisConn:
```

Comment:
Need to revisit this and `FakePool`. The whole thing is starting to become a mess and hard to understand. Maybe its worth to just separate in 2 classes. One for each version of redis and then let the fixture decide when to load one or the other